_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+
+
+
+* Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow GitHub Page](https://github.com/kubeflow)
+* Join the [Kubeflow Slack channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+* Join the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list
+* Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/)
+
+
+## Component Versions
+
+
+<table>
+  <tr>
+   <td><strong>Maintainers</strong>
+   </td>
+   <td><strong>Component Name</strong>
+   </td>
+   <td><strong>Version</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>AutoML Working Group
+   </td>
+   <td>Katib
+   </td>
+   <td><a href="https://github.com/kubeflow/katib/issues/2156">v0.16.0</a>
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="9" >Notebooks Working Group
+   </td>
+   <td>Admission Webhook (PodDefaults)
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Central Dashboard
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Jupyter Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Kubeflow Access Management API
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Notebook Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Profile Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Tensorboard Controller
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Tensorboard Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td>Volumes Web App
+   </td>
+   <td>v1.8.0
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="2" >Pipelines Working Group
+   </td>
+   <td>Kubeflow Pipelines
+   </td>
+   <td><a href="https://github.com/kubeflow/pipelines/releases/tag/2.0.2">v2.0.3</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Kubeflow Pipelines Tekton
+   </td>
+   <td><a href="https://github.com/kubeflow/manifests/pull/2500">2.0.3</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Serving Working Group
+   </td>
+   <td>KServe
+   </td>
+   <td><a href="https://github.com/kserve/kserve/releases/tag/v0.11.1">v0.11.1</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Training Working Group
+   </td>
+   <td>Training Operator
+   </td>
+   <td><a href="https://github.com/kubeflow/training-operator/releases/tag/v1.7.0-rc.0">v1.7.0</a><span style="text-decoration:underline;">-rc0</span>
+   </td>
+  </tr>
+</table>
+
+
+
+### Dependency Versions (Manifests)
+
+
+<table>
+  <tr>
+   <td><strong>Component</strong>
+   </td>
+   <td><strong>Version used in 1.8</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://knative.dev/docs/reference/relnotes/">Knative</a> 
```

Comment:
nit: instead of having a link to the dependency, would it be better to link the versions like Kubeflow components